projects/mod_2_homework/src/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+typedef struct {
```

Comment:
The usual convention in our codebase is to put the name of the struct both here (i.e. `typedef struct Counters {`) and in the typedef, it helps with IDE name resolution.